testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
I'm assuming by 'we have code that checks for the existense of a private keyword' ... 
you mean: IDEA Haxe plugin implementation (in bnf) that exposes 'private' occurence and the code that checks its existense? 

Or are you referring to something in our (TiVo) application (HDUI) code?

If it is the 1st case... I need some help in understanding what will break there - so I can guess what I need to find and fix